docs/guides/0-web-verification.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Android → Web Verification
+sidebar_position: 1
+---
+
+# Enable Web Credential Sharing in Your Android App
+
+This guide explains how to implement the W3C Digital Credentials API into the Multipaz Getting Started Sample app using the Multipaz SDK. The integration enables your app to present credentials to web-based verifiers using the standardized Credential Manager interface, including privileged browser and user agent support.
+
+## **Overview**
+
+The W3C DC API integration allows your app to interact with web-based verifiers via the Credential Manager API, supporting secure and privacy-preserving credential presentment flows. With this, users can share credentials directly with browsers that support the API. To implement this using Multipaz SDK, these steps are required:
+
+* Implementing a new `CredmanActivity` to handle credential requests.
+* Registering privileged user agents (browsers) via a `privilegedUserAgents.json` file.
+* Updating the app initialization to support exporting credentials through the API.
+
+## **How It Works**
+
+When a user visits a web verifier (like [verifier.multipaz.org](https://verifier.multipaz.org)), here's what happens:
+
+1. **Browser Request**: The verifier website uses the W3C Digital Credentials API to request credentials
+2. **Android Response**: Your app (as a credential provider) receives the request and shows the credential selection UI
+3. **Server Verification**: The verifier server processes and verifies the returned credentials
+4. **Result Display**: The browser displays the verified credential information
+
+The verifier server handles the complex cryptographic verification, while your Android app focuses on secure credential presentation.
+
+## **Implementation Steps**
+
+### **1. Add CredmanActivity**
+
+Create a new activity extending `CredentialManagerPresentmentActivity `provided by Multipaz. This activity is launched when a browser requests credentials via the W3C DC API.
+
+```
```

Comment:
These snippets don’t seem to show Kotlin syntax highlighting. Could it be because the code block isn’t marked with `kotlin` at the beginning?

````
```kotlin
````